src/main/java/com/lambdaschool/oktafoundation/controllers/ClubActivityController.java

```diff
@@ -107,7 +105,7 @@
                          *  Here we just add the reactionint for the reaction given */
                         if(ar.getActivityid() == clubActivityFeedback.getActivityid()) {
                             activityIdFound = true;
-                            Integer reactionIntval = reactionService.findReactionById(clubActivityFeedback.getReactionid()).getReactionint();
+                            Double reactionIntval = reactionService.findReactionById(clubActivityFeedback.getReactionid()).getReactionint();
```

Comment:
what do you think about changing the getReactionInt name to getReactionDouble to make the code more intuitive?

Reply:
I agree. You are right.